sdk/digitaltwins/azure-digitaltwins-core/pom.xml

```diff
@@ -129,6 +129,7 @@
             <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
             <configuration>
               <argLine>
+                --add-exports com.azure.core/com.azure.core.implementation.http=ALL-UNNAMED
```

Comment:
missing piece from the previous PR, if you want to run tests live/record you need this if you are using JDK 9+